docs/guides/using-theme-overrides.md

```diff
@@ -36,11 +39,17 @@ embed: true
 
 ### How theming works in InstUI
 
-The theming system in InstUI has two levels:
+The theming system in InstUI has three levels:
 
 **The global theme**
 
-On the broader level, there is the main theme object that contains the color, spacing, typography etc. variables available in the theme (e.g.: [canvas theme](/#canvas)). The global theme can be set via the [InstUISettingsProvider](/#InstUISettingsProvider) component.
+Global themes are useful when you have multiple React Application trees and you want to set up themes and overrides on a more global level than application level theming.
+This basically means you don't necesseraly have to warp each application tree with an [InstUISettingsProvider](/#InstUISettingsProvider) to use themes.
```

Comment:
typo: ... **necessarily** have to **wrap**...

Reply:
fixed